### PR TITLE
Show policies pull2 [P0]

### PR DESCRIPTION
**Pull request by khushi2305**
This page shows the policies available for the user ( depending on the pincode ) .
Things to be reviewed -
1)Only one of the policies can be selected and chosen policy is saved in a local variable ( later will be stored in the User class if payment done successfully ) .
3) In case the user exits the screen , he will be directly navigated back to the home screen .
4) The policy list is made scrollable and therefore any number of policies can be added to the by the company . 
2) Buttons have been connected to their respected pages . 
3) Formatting and UI has been improved and widgets are made screen size compatible .

Show_Policies

### CODE REVIEW

**Comment by khushi2305** [comment]
@googlebot I consent.

File: home_insurance_app/lib/pages/choose_policy.dart [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                  },
+                  backgroundColor: Colors.lightBlueAccent,
+                  icon: Icon(Icons.payment),
+                  label: Text(
+                    'View Smart Device Discounts',
+                    style: TextStyle(
+                      color: Colors.black54,
+                      fontSize: 15.0,
+                      fontWeight: FontWeight.bold,
+                    ),
```
[the remark this answers is not in the document]
Comment (khushi2305) [reply home_insurance_app/lib/pages/choose_policy.dart]: Done

File: home_insurance_app/lib/pages/choose_policy.dart [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                  backgroundColor: Colors.lightBlueAccent,
+                  icon: Icon(Icons.payment),
+                  label: Text(
+                    'Payment',
+                    maxLines: 2,
+                    style: TextStyle(
+                      color: Colors.black54,
+                      fontSize: 15.0,
+                      fontWeight: FontWeight.bold,
+                    ),
```
[the remark this answers is not in the document]
Comment (khushi2305) [reply home_insurance_app/lib/pages/choose_policy.dart]: Done

File: home_insurance_app/lib/pages/choose_policy.dart [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  int index;
+  Mapping(int index, Policy policyOption) {
+    this.index = index;
+    this.policyOption = policyOption;
+  }
+}
+
+Policy userChoice; // Stores the policy selected by the user
+
+class DisplayPolicies extends StatefulWidget {
```
[the remark this answers is not in the document]
Comment (khushi2305) [reply home_insurance_app/lib/pages/choose_policy.dart:19]: Done

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

File: home_insurance_app/lib/pages/show_discounts.dart [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+class DisplayDiscounts extends StatefulWidget {
```
Comment (dreamplay) [inline home_insurance_app/lib/pages/show_discounts.dart:3]: Add comment

File: home_insurance_app/lib/pages/style/custom_widgets.dart [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+Divider CustomDivider({double height = 0, double width = 0}) {
```
Comment (dreamplay) [inline home_insurance_app/lib/pages/style/custom_widgets.dart:3]: Add comment

**Comment by khushi2305** [comment]
This commit displays all the available policies to the user depending upon their pin-code and user can select one of the policy .

**Pull request merged**